translation_en/chapter_02_PT.md

```diff
@@ -27,7 +27,7 @@ Enquanto costumávamos ter um monólito com todos os recursos de nossas grandes
 
 Construir um serviço hoje com endpoints REST é uma tarefa bastante fácil se você estiver usando um desses frameworks mencionados anteriormente. Você tem um modelo de programação baseado em anotação que permite mapear métodos Java para terminais REST e mecanismos avançados de serialização/desserialização que lidarão com todo o boilerplate de análise de solicitações HTTP.
 
-> **DICA:** Para mais detalhes sobre a arquitetura de microsserviços, consulte o capítulo [Microsserviços] () #TODO na fase de revisão: adicionar referência a https://github.com/otaviojava/manual-arquiteto-moderno/blob/master/capítulo_05.md
+> **DICA:** Para mais detalhes sobre a arquitetura de microsserviços, consulte o capítulo [Microsserviços](https://github.com/betrybe/manual-arquiteto-moderno/blob/master/chapter_07.md).
```

Comment:
Quando esse conteúdo for convertido a formato de livro online, esse link deveria ser uma ancora para o capitulo sete, não para o github. Será que seria possível esse tipo de ancora no conteúdo final?

Reply:
Se colocar apenas "chapter_07.md" funciona